strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java

```diff
@@ -152,9 +154,9 @@ public URI toUri()
         throw new UnsupportedOperationException();
     }
 
-    public Path toAbsolutePath()
+    public RepositoryPath toAbsolutePath()
     {
-        throw new UnsupportedOperationException();
+        return createByTemplate(getTarget().toAbsolutePath());
```

Comment:
1) Could you please rename `createByTemplate` to `createFromTemplate`?
2) I'm not sure I understand what this is doing? Would you mind explaining?